dashboard/db/migrate/20171205010302_create_featured_projects.rb

```diff
@@ -0,0 +1,12 @@
+class CreateFeaturedProjects < ActiveRecord::Migration[5.0]
+  def change
+    create_table :featured_projects do |t|
+      t.integer :storage_app_id
+      t.integer :who_featured_user_id
+      t.integer :whose_project_user_id
+
+      t.timestamps
```

Comment:
Didn't we decide we only wanted `created_at` not `updated_at`? If so, instead of `t.timestamps`, create the desired column directly:

```
t.datetime :created_at
```